turbo.json

```diff
@@ -17,6 +17,7 @@
       "outputs": ["lib/**", "index.js"],
       "dependsOn": ["^build"]
     },
-    "lint": {}
+    "lint": {},
+    "//#check:format": {}
```

Comment:
It's an actual key it's a special format to target the parent's tasks instead of going to each package. 
ref: https://turbo.build/repo/docs/core-concepts/monorepos/filtering#the-workspace-root
https://github.com/vercel/turbo/pull/1260/files#diff-68d3a640fedd680754f4aa3264db86ca921a65a71be6108e80e8b933e83b0b32R257